### PR TITLE
fix: Baguette (french) translations

### PR DESCRIPTION
**Pull request by Rushaway**
https://github.com/orgs/alliedmodders/projects/1/views/1?pane=issue&itemId=24307187

### CODE REVIEW

File: translations/fr/nominations.phrases.txt [hunk 1]
```diff
@@ -65,4 +65,9 @@
 		"fr"		"Sélectionnée"
 	}
 
+	"Map Not In Pool"
+	{
+		"fr"		"La map '{1}' n'est pas dans la piscine de nominations."
```
Comment (Kenzzer) [inline translations/fr/nominations.phrases.txt:70]: `Piscine` is way too of literal translation of `pool`. Consider `liste` instead or another non literal translation word that means selection/list, not a literal swimming pool.

Also is it really `de nominations`, feels like it should be `des nominations` in french. IIRC you can only use `de` for a plural word when the given word doesn't change in the singular tense.

**Review by Kenzzer — COMMENTED** [review]
Can confirm the translations as a french individual. But I do have one nitpick about `Map Not In Pool`.

File: translations/fr/nominations.phrases.txt (hunk 1 above)
Comment (Rushaway) [reply translations/fr/nominations.phrases.txt]: It's true. Merci à vous. :)

**Review by asherkin — APPROVED** [review]
Thanks!

**Pull request merged**